subjects/forum/forum.en.md

```diff
@@ -0,0 +1,101 @@
+## forum
+
+### Objectives
+
+This project consists in creating a web forum that allows :
+
+- communication between users and the community through the creation of posts/comments.
+- associate posts to categories.
+- non-registered users to only see posts/comments.
+- likes and dislikes in posts and comments, you must take in consideration that the number of likes and dislikes must be accounted.
+- searching, inside your forum, for a specific category or post
+  - The search bar must have typing suggestions as you write.
```

Comment:
I don't think type suggestions as you type can be done without JS, would not ask for it

Reply:
It's possible using the datalist from html and go templates, but i prefer to remove it and do it for the next forum? because in JS its better